### DISCUSSION TITLE
2.3.6 HomeAssistant MQTT Question

### DISCUSSION THREAD

**Discussion by jmp2000** [aqualinkd/AqualinkD#309] [Q&A] [answered]
With the (awesome) new 2.3.6 version - do we need to put all the Home Assistant configuration.yaml text shown in: 

https://github.com/sfeakes/AqualinkD/blob/master/extras/HASSIO.Implementation.txt

Or does the new settings take care of it automatically (for a default configuration in Home Assistant using MQTT) ?

**Comment by sfeakes** [comment, marked as answer]
If you are using the new home assistant MQTT discovery feature with AqualinkD, then no their is no need to use that yaml file.
You simply set the home assistant discovery topic in aqualinkd.conf.  (Also making sure home assistant is using the same MQTT broker as aqualinkd).
